examples/Table-icons.py

```diff
@@ -68,14 +75,23 @@ def main(self):
 
     # Read a directory with icons.
     #
-    def readIconDir(dir)
-      iconList = SCR.Read(Path(".target.dir"), dir).asList()
+def readIconDir(dir):
+      iconList = list(SCR.Read(Path(".target.dir"), dir))
       ycpbuiltins.y2debug("Dir %1: %2  entries", dir, ycpbuiltins.size(iconList))
-      iconList = ycpbuiltins.sort(ycpbuiltins.filter(iconList) do |entry|
-        ycpbuiltins.regexpmatch(entry, "^.*.(png|jpg|PNG|JPG)$")
-      end)
-
-      copy.deepcopy(iconList)
+#      #TODO #FIXME add ycbbuiltins.filter()
+#      #TODO #FIXME add ycbbuiltins.sort()
+#      iconList = ycpbuiltins.sort(ycpbuiltins.filter(iconList) do |entry|
+#        ycpbuiltins.regexpmatch(entry, "^.*.(png|jpg|PNG|JPG)$")
+#      end)
```

Comment:
for me it looks like both TODO and FIXME is done, not? then we can remove this comments

Reply:
nope, these still aren't implemented (see src/ycpbuiltins.py)

Reply:
ah, ok.